### PR TITLE
add chipsea csm64f02 & csm64f02_b

### PR DESCRIPTION
**Pull request by Samuel-0-0**
add chipsea csm64f02 with 2mb flash & csm64f02_b with 1mb flash

### CODE REVIEW

**Comment by Samuel-0-0** [comment]
add chipsea csm64f02 with 2mb flash & csm64f02_b with 1mb flash

**Comment by Samuel-0-0** [comment]
add chipsea csm64f02 with 2mb flash & csm64f02_b with 1mb flash

**Comment by jihao** [comment]
This is useful, can someone merge this please?

File: boards/csm64f02.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  ],
+  "name": "CHIPSEA CSM64F02",
+  "upload": {
+    "maximum_ram_size": 81920,
+    "maximum_size": 2097152,
+    "require_upload_port": true,
+    "resetmethod": "nodemcu",
+    "speed": 115200
+  },
+  "url": "http://www.esp8266.com/wiki/doku.php?id=esp8266-module-family",
```
Comment (pfeerick) [inline boards/csm64f02.json]: Should probably be http://www.chipsea.com/en/wifi/CSM64F02.html, not the espressif documentation

File: boards/csm64f02.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  "vendor": "Espressif"
```
Comment (pfeerick) [inline boards/csm64f02.json]: ... and the module is by Chipsea Technologies...

File: boards/csm64f02_b.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  ],
+  "name": "CHIPSEA CSM64F02_B 1M",
+  "upload": {
+    "maximum_ram_size": 81920,
+    "maximum_size": 1048576,
+    "require_upload_port": true,
+    "resetmethod": "nodemcu",
+    "speed": 115200
+  },
+  "url": "http://www.esp8266.com/wiki/doku.php?id=esp8266-module-family",
```
Comment (pfeerick) [inline boards/csm64f02_b.json]: http://www.chipsea.com/en/wifi/CSM64F02.html

File: boards/csm64f02_b.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  "vendor": "Espressif"
```
Comment (pfeerick) [inline boards/csm64f02_b.json]: Chipsea Technologies

**Pull request closed without merging**